src/app/ui/earn/EarnFilteringContext.tsx

```
@@ -116,24 +149,65 @@ export const EarnFilteringProvider = ({
+    const endIndex = startIndex + PAGE_SIZE;
+    const paginatedData = data.slice(startIndex, endIndex);
+
+    let updatedAt: Date | undefined;
+    let error: unknown | undefined;
+
+    switch (tab) {
+      case EarnFilterTab.ALL:
+      case EarnFilterTab.YOUR_POSITIONS: {
+        updatedAt = all.data?.meta?.updatedAt;
+        error = all.error;
+      }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)